src/main/java/frc/robot/Robot.java

```diff
@@ -139,6 +166,13 @@ public void teleopPeriodic() {
     selectColor();
     selectControlPanelSpinAmount();
     colorDetector();
+
+    // sensor returns a value from 0-4095 that is scaled to inches
```

Comment:
For consistency with the other sentence comments, this should follow grammatical conventions.